lib/addons/src/postinstall/codemods.ts

```diff
@@ -0,0 +1,50 @@
+interface PostinstallContext {
+  root: any;
+  api: any;
+}
+
+export function addPreset(preset: string, presetOptions: any, { api, root }: PostinstallContext) {
```

Comment:
Why is this part of addons? It seems not meant to be part of either the manager or preview's code at all?

Reply:
Didn't want add-ons to have to take that dependency. You'll notice that this uses dependency injection to avoid needing any extra deps